### PR TITLE
Add identifier RAiD as owl:Class in details.ttl

### PR DESCRIPTION
**Pull request by andand79**

### CODE REVIEW

**Review by niklasl — COMMENTED** [review]
Looks good! Could you also add some definition reference (see uses of `rdfs:seeAlso` and `skos` matches on other `Identifier` subclasses)?

(As a follow-up we may go trough the existing ones and perhaps change some chosen relationships (e.g. to `dct:references`).)

**Comment by andand79** [comment]
With the identifier being new, no other exactMatch have been found. subClassOf changed to URI since the identifiers are said to be expressed as actionable URLs.

File: source/vocab/details.ttl [hunk 1]
```diff
@@ -694,7 +694,8 @@
     :category :pending ;
     rdfs:label "RAiD";
     rdfs:comment "Research Activity Identifier";
-    rdfs:subClassOf :Identifier .
+    skos:definition "Persistent identifier for research projects; governed by ISO 23527 and managed by Australian Research Data Commons."@en;
```
Comment (niklasl) [inline source/vocab/details.ttl:697]: Add e.g.:
```
    rdfs:seeAlso <https://raid.org/> ;
```
if that's the official site for its definition.

**Review by niklasl — APPROVED** [review]
LGTM!

**Comment by andersju** [comment]
I assume we can go ahead and merge this? :)

**Comment by andand79** [comment]
Yes! At the moment I'm not authorised to merge so please go ahead.

**Pull request merged**